clients/drdisas/drdisas.cpp

```diff
@@ -41,25 +41,18 @@ namespace {
 // XXX i#1684: We want cross-arch decoding support so a single build can decode
 // AArchXX and x86.  For now, a separate build is needed.
 #ifdef X86
-droption_t<std::string>
-    op_syntax(DROPTION_SCOPE_FRONTEND, "syntax", "dr_default",
-              "Uses the specified syntax: 'dr_default', 'intel' or 'att'.",
-              "Uses the specified syntax: 'dr_default', 'intel' or 'att'.");
 #    ifdef X86_64
 droption_t<std::string> op_mode(DROPTION_SCOPE_FRONTEND, "mode", "x64",
                                 "Decodes using the specified mode: 'x64' or 'x86'.",
                                 "Decodes using the specified mode: 'x64' or 'x86'.");
 #    endif
-#elif defined(AARCH64) || defined(ARM)
-droption_t<std::string> op_syntax(DROPTION_SCOPE_FRONTEND, "syntax", "dr_default",
-                                  "Uses the specified syntax: 'dr_default'' or 'arm'.",
-                                  "Uses the specified syntax: 'dr_default' or 'arm'.");
-
-#    if defined(ARM)
+droption_t<std::string> op_syntax(DROPTION_SCOPE_FRONTEND, "syntax", "intel",
+                                  "Uses the specified syntax: 'intel', 'att' or 'dr'.",
+                                  "Uses the specified syntax: 'intel', 'att' or 'dr'.");
+#elif defined(ARM)
```

Comment:
I would expect a comment explaining why there is no syntax option, so the reader doesn't have to puzzle over it.

Reply:
For now, I just added a comment referring to the issue page (indeed atm without an elaborate root cause analysis). Does this suffice for now?

Reply:
Just sthg indicating that we have a reason for this and it wasn't a mistake.

Reply:
Done.